kafka-net-core/Producer.cs

```diff
@@ -270,6 +270,10 @@ private async Task ProduceAndSendBatchAsync(List<TopicMessage> messages, Cancell
                         MessagesSent = group.Select(x => x.TopicMessage).ToList()
                     };
 
+                    /* TODO: Build warning: [CS4014] Because this call is not awaited, execution of the current
+                     method continues before the call is completed. Consider applying the 'await' operator to the
+                     result of the call. */
+                    
```

Comment:
suppress it please with this code and remove todo comment
```
#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
                    brokerSendTask.Task.ContinueWith(t => { _semaphoreMaximumAsync.Release(); }, cancellationToken);
#pragma warning restore CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
```

This code is without await on purpose.

Reply:
I'm sorry, but disabling this warning seems wrong - it's there for a reason! ;)
My pull request is related to supporting .net standard and not fixing existing code so this is outside of the request's scope.

I think you need to re-look at your code and understand where is best to await this Task.
Best regards,

Reply:
If the scope of the task is supporting ,net standard and you don't agree with my comment so just discard changes in this file as it's not related to the described scope.